sass/_FullSearchGrid.scss

```diff
@@ -8,13 +8,28 @@ $gutter: 0em;
 $page-width: 1000px;
 
 .CoveoSearchInterface {
-
   min-width: 300px;
 
+  &.coveo-waiting-for-query {
+    > * {
+      display: none;
+      visibility: hidden;
```

Comment:
It's more of a shotgun approach so that everything is hidden properly, no matter what.

For example, some components will change their appearance on page resize, and the CSS gets more specific and artifact could show.

By setting both property, it's more likely that I am not forgetting any weird components exception.